lib/interpreters/split_clustering.rb

```diff
@@ -257,14 +260,23 @@ def self.split_solve_core(service_vrp, job = nil, &block)
         # RECURSIVELY CALL split_solve_core AND MERGE SOLUTIONS
         solutions = sides.collect.with_index{ |side, index|
           service_vrp[:split_level] = split_level + 1
+          service_vrp[:split_denominators] << 2**service_vrp[:split_level]
+          service_vrp[:split_sides] << index
```

Comment:
I remember discussing basing this on the number of services or the optimization duration, was that going to happen in another dev?

Reply:
Yes, it will be done in another dev